horizon_autotests/steps/volumes.py

```diff
@@ -384,7 +384,7 @@ def tab_backups(self):
 
     @pom.timeit('Step')
     def create_snapshot(self, volume_name, snapshot_name, description=None,
-                        check=True):
+                        check=True, modal_absent=True):
```

Comment:
In case of creating subnet with same name we shouldn't check that it present on page, but should wait until form to be closed.